object/adapter.go

```diff
@@ -19,7 +19,8 @@ import (
 	"runtime"
 
 	"github.com/astaxie/beego"
-	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/go-sql-driver/mysql" // mysql
+	_ "github.com/lib/pq"              // postgres
```

Comment:
comment pg

Reply:
Users should fill `app.conf` with `db = "postgres"`, or xorm can't connect to pg. So I suggest to use `postgres` here.